### PR TITLE
Nikolay484

### PR DESCRIPTION
**Pull request by nikolay484**

### CODE REVIEW

File: public/class-fontimator-myaccount.php [hunk 1]
```diff
@@ -716,7 +716,7 @@ public function add_address_field_to_edit_account () {
         if ( ! $user_address ) {
             $user_address = new stdClass();
         }
-        if ( ! $user_address->country ) {
+        if (isset($user_address->country) ) {
```
Comment (reuvenkarasik) [inline public/class-fontimator-myaccount.php:719]: Maybe I'm wrong, but this looks like a mistake :)

Comment (nikolay484) [reply public/class-fontimator-myaccount.php:719]: Undefined property: stdClass::$country in plugins/fontimator/public/class-fontimator-myaccount.php on line 719 

you can offer another solution

Comment (reuvenkarasik) [reply public/class-fontimator-myaccount.php:719]: No, I just meant that `! $variable` is the opposite of `isset($variable)`; I just think a `!` needs to be added :)

Comment (nikolay484) [reply public/class-fontimator-myaccount.php:719]: in php 8.1 it throws warning, in 8.2 will throw fatal exception

**Comment by avrahamcornfeld** [comment]
I added the ! back

**Pull request merged**